### PR TITLE
Implement empty function

### PR DESCRIPTION
**Pull request by jshin1394**
Adds empty function to metrax metrics that directly inherit from `clu.metric`. Note how `clu.Average` already implements empty fuction.

### CODE REVIEW

File: src/metrax/metrics.py [hunk 1]
```diff
@@ -107,6 +107,15 @@ class RSQUARED(clu_metrics.Metric):
...
   sum_of_squared_label: jax.Array
+  
+
+  @classmethod
+  def empty(cls) -> 'RSQUARED':
+    return cls(
+      total=jnp.array(0, jnp.float32),
+      count=jnp.array(0, jnp.float32),
+      sum_of_squared_error=jnp.array(0, jnp.float32),
+      sum_of_squared_label=jnp.array(0, jnp.float32))
```
Comment (jeffcarp) [inline src/metrax/metrics.py:118]: Thanks! Would you mind adding a quick test for here and below?

Comment (jshin1394) [reply src/metrax/metrics.py:118]: added testcases for each class.

**Comment by jeffcarp** [comment]
Thank you!
